Lunch/TodaysLunchViewController.swift

```diff
@@ -7,20 +7,150 @@
 //
 
 import UIKit
+import Parse
+import MapKit
+import GoogleMaps
 
-class TodaysLunchViewController: UIViewController {
+class TodaysLunchViewController: UIViewController, MKMapViewDelegate {
+
+    class CustomPointAnnotation: MKPointAnnotation {
+        var imageName: String!
+    }
+
+    @IBOutlet weak var selectedPlaceLabel: UILabel!
+    @IBOutlet weak var selectedVotesLabel: UILabel!
+    @IBOutlet weak var mapView: MKMapView!
 
-    @IBOutlet weak var todaysLunchLabel: UILabel!
-    
     var textForLunchLabel = ""
-    
+    var textForVotesLabel = ""
+    var selectedIndex = -1
```

Comment:
Optionals are a more semantic way of indicating that there's no value yet. `var selectedIndex: Int?`